docs/linux.md

```diff
@@ -141,3 +141,8 @@ sudo rm -r /usr/share/ollama
 sudo userdel ollama
 sudo groupdel ollama
 ```
+
+Remove the config
+```bash
+sudo rm -f /etc/ollama
```

Comment:
should be `sudo rm -rf /etc/ollama`. I test on my VM and get output:
```shell
sudo rm -f /etc/ollama
rm: cannot remove '/etc/ollama': Is a directory
```

Reply:
you are right